sigs.yaml

```diff
@@ -81,7 +89,7 @@ sigs:
             - https://raw.githubusercontent.com/kubevirt/kubevirt.github.io/main/OWNERS
             - https://raw.githubusercontent.com/kubevirt/user-guide/main/OWNERS
             - https://raw.githubusercontent.com/kubevirt/kubevirt-tutorial/main/OWNERS
-    - name: storage
+    - name: sig-storage
```

Comment:
nit: since this is inside the `sigs:` section the sig repetition feels kind of redundant, however I don't feel strongly either way.

Reply:
Thanks for explaining the reasoning behind this! I've missed the sigs section :)

I mainly tried to keep things consistent. Would you prefer if I'd remove the `sig-` prefix from compute?

Reply:
It's probably better if we do drop the sig prefix from that column, mostly just to make the label clearer. With the prefix, it's a single character difference.

Reply:
Done! :)
PTAL